pkg/options/options.go

```diff
@@ -24,12 +33,13 @@ type Hub struct {
 
 // Define the shape of clusters that may be added under management
 type ManagedClusters struct {
-	ConfigDir string `yaml:"configDir,omitempty"`
+	ConfigDir       string `yaml:"configDir,omitempty"`
+	Owner           string `yaml:"owner,omitempty"`
```

Comment:
what is this owner? indicating that it's a hive cluster created by a hub?

Reply:
so it's the prefix of a cluster?

Reply:
yes... and can be overwritten by a parameter in the cmd and if it is not specify in the cmd nor options.yaml it will be the $USER